package.json

```diff
@@ -33,6 +33,7 @@
     "jsonwebtoken": "^8.5.1",
     "mongoose": "^6.2.3",
     "morgan": "^1.10.0",
+    "multer": "^1.4.4",
     "nodemon": "^2.0.15"
   }
 }
```

Comment:
we are saving images to file system not to database ?

Reply:
shall we do both? database and file system? I thought about storing images as source reference in database but I will change it to actual image.